gradle/plugins/src/main/kotlin/com.hedera.gradle.examples.java.gradle.kts

```diff
@@ -77,9 +77,6 @@ abstract class RunAllExample : DefaultTask() {
             .asSequence()
             .map { it.name.replace(".java", "") }
             .filter { it != "ValidateChecksumExample" } // disabled this example, because it needs user input (but it WORKS)
-            .filter { it != "ZeroTokenOperationsExample" } // doesn't work
-            .filter { it != "SolidityPrecompileExample" } // doesn't work with hedera-local-node
-            .filter { it != "ConsensusPubSubChunkedExample" } // doesn't work on CI
```

Comment:
should revert the pub sub i think

Reply:
@thenswan Do you want to skip this example for now and merge this PR. After it's fixed in local-node and has consistent behaviour we can enable it.